clash-cores/src/Clash/Cores/Floating/Xilinx.hs

```diff
@@ -0,0 +1,104 @@
+{-|
+Copyright  :  (C) 2021,      QBayLogic B.V.,
+License    :  BSD2 (see the file LICENSE)
+Maintainer :  QBayLogic B.V. <devops@qbaylogic.com>
+
+Support for the [Xilinx Floating-Point LogiCORE IP v7.1](https://www.xilinx.com/support/documentation/ip_documentation/floating_point/v7_1/pg060-floating-point.pdf).
+
+The functions in this module make it possible to use the Xilinx IP in Clash.
+Compilation will instantiate the Xilinx IP. Clash will output a TCL script
+named @floating_point@/.../@.tcl@ which needs to be executed in the Vivado
+project to create the proper entity. Simulation in Clash produces bit-identical
+results to synthesis.
```

Comment:
Is this comment up-to-date? I remember some discussion on Slack, but didn't actively follow it.

Reply:
It's correct for addition. A few unary operators have a 1 ULP (unit in last place) error in Xilinx, which I don't think we can simulate without knowing how the Xilinx IP works (it's "encrypted"). The plan is to add this to this documentation when those operators are added.

Reply:
Oh right, we **can** simulate it bit-accurate, but it requires a foreign function interface and then using Xilinx's C simulation shared lib. Or perhaps the portion of that library that contains the necessary information is even open source, which would enable us to take a close look at it. I haven't looked at the C sim lib from Xilinx yet.

Reply:
Oh yeah, don't bother with that IMO.